packages/generator/src/edmx-parser/legacy/common/edmx-to-vdm.ts

```diff
@@ -110,7 +110,7 @@ export function transformEntity(
   classNames: Record<string, any>,
   complexTypes: VdmComplexType[],
   formatter: ServiceNameFormatter
-): Omit<VdmEntity, 'navigationProperties'> {
+): Omit<Omit<VdmEntity, 'navigationProperties'>, 'entityTypeNamespace'> {
```

Comment:
I think you can do something like: `Omit<VdmEntity, 'navigationProperties' | 'entityTypeNamespace'>`. Or it is `&` instead of `|`. For types the union is `|` but for object the `&` so I do not know which one to use here.